go-controller/pkg/ovn/egressip.go

```diff
@@ -893,13 +900,16 @@ func (oc *Controller) syncStaleEgressReroutePolicy(egressIPToPodIPCache map[stri
 func (oc *Controller) syncStaleNATRules(egressIPToPodIPCache map[string]sets.String) {
 	predicate := func(item *nbdb.NAT) bool {
 		egressIPName, exists := item.ExternalIDs["name"]
-		// Skip nat rows that do not have egressIPName attribute available
-		if !exists {
+		// Exclude rows that have no name or are not the right type
+		if !exists || item.Type != nbdb.NATTypeSNAT {
```

Comment:
Took me a while to figure out why SNAT. I have the feeling these methods shouldbe named after that: `syncStaleSNATRules`. Just a nit.

Reply:
indeed; egressip uses snat only.